app/daos/UserDaoMongoDBImpl.js

```diff
@@ -22,7 +22,20 @@ module.exports = function() {
     });
   };
 
+  const signin = function(userData) {
```

Comment:
Changed function name to reflect `getByEmailOrUsername`. Also will get user record by email or username.